src/modules/endpoint/sagas/watchFetchEvent.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import type { Saga } from 'redux-saga'
+import { call, put, takeLatest } from 'redux-saga/effects'
+import type { FetchEventActionType, FetchEventFailedActionType, PushEventActionType } from '../../app/StoreActionType'
+import MemoryDatabase from '../MemoryDatabase'
+import loadCityContent from './loadCityContent'
+
+function * fetchEvent (database: MemoryDatabase, action: FetchEventActionType): Saga<void> {
+  const {city, language, pushParams} = action.params
+  try {
+    yield call(loadCityContent, database, city, language)
+
+    const insert: PushEventActionType = {
+      type: `PUSH_EVENT`,
```

Comment:
Should we maybe change this to PUSH_CITY_CONTENT?
Because this code is very similar to watchFetchCategory

Reply:
No, the reducers will do very different stuff.